nilearn/_utils/numpy_conversions.py

```diff
@@ -124,3 +126,46 @@ def as_ndarray(arr, copy=False, dtype=None, order='K'):
         raise ValueError("Type not handled: %s" % arr.__class__)
 
     return ret
+
+
+def csv_to_array(csv_path, delimiters=' \t,;', **kwargs):
+    """Read a CSV file by trying to guess its delimiter
+
+    Parameters
+    ----------
+
```

Comment:
You are fast, Sir.

2015-07-20 0:14 GMT+02:00 Alexandre Abraham <notifications@github.com>:

> In nilearn/_utils/numpy_conversions.py
> <https://github.com/nilearn/nilearn/pull/703#discussion_r34962871>:
>
> > @@ -124,3 +126,46 @@ def as_ndarray(arr, copy=False, dtype=None, order='K'):
> >          raise ValueError("Type not handled: %s" % arr.__class__)
> >
> >      return ret
> > +
> > +
> > +def csv_to_array(csv_path, delimiters=' \t,;', **kwargs):
> > +    """Read a CSV file by trying to guess its delimiter
> > +
> > +    Parameters
> > +    ----------
> > +
>
> Fixed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/703/files#r34962871>.
>



-- 
Viele Grüße,
Danilo